src/System.ComponentModel.Annotations/ref/System.ComponentModel.Annotations.cs

```diff
@@ -226,6 +226,7 @@ public partial class RequiredAttribute : System.ComponentModel.DataAnnotations.V
     {
         public RequiredAttribute() { }
         public bool AllowEmptyStrings { get { throw null; } set { } }
+        public bool AllowEmptyCollections { get { throw null; } set { } }
```

Comment:
I don't see any api-approved issues for this property.  If you're trying to get new API added, it needs to follow the API Review/Approval process: https://github.com/dotnet/corefx/blob/master/Documentation/project-docs/api-review-process.md